src/Native/WebGL.js

```diff
@@ -226,10 +229,10 @@ var _elm_community$webgl$Native_WebGL = function () {
     var dataIdx = 0;
     var array = new attributeInfo.type(listLength(bufferElems) * attributeInfo.size * elemSize);
 
-    A2(listMap, function (elem) {
+    A3(listFoldl, F2(function (elem) {
       dataFill(array, attributeInfo.size, dataIdx, elem, attribute.name);
       dataIdx += attributeInfo.size * elemSize;
-    }, bufferElems);
+    }), undefined, bufferElems);
```

Comment:
it would be better to just do this by hand rather than using list methods in an impure way.